### DISCUSSION TITLE
Discord connected

### DISCUSSION THREAD

**Discussion by ENVY1404** [pkgxdev/pkgx#918] [Help] [answered]
how to connect discord with github?

**Comment by wasajang** [comment]
i want to know it as well

**Comment by Biginsulin-stack** [comment]
Just sign up on github if you haven't done it before.. then connect ur discord

**Comment by DaveiV** [comment]
just connect by seting on discord setting

**Reply by DropHunterXYZ** [reply]
Yup thats it!
Maybe connect the twitter (X) as well

**Comment by boredray** [comment, marked as answer]
Click User Settings > click Connections > Click GitHub > Follow the prompts on GitHub. 

hope it helps!
